ADOPTERS.md

```diff
@@ -77,5 +77,5 @@
-| [Overwolf](https://www.overwolf.com)                                         | [@tomwolfgang](https://github.com/tomwolfgang)                                                                                                                                                                           | Dev portal - software catalog, tech-docs, scaffolding                                                                                                                                                                                                               |
-| [Hotmart](https://www.hotmart.com)                                         | [@fabioviana-hotmart](https://github.com/fabioviana-hotmart)                                                                                                                                                                           | The main Developers Portal to centralize docs, applications and technical metrics.                                                                                                                                                                                                               |
+| [Overwolf](https://www.overwolf.com)                                     | [@tomwolfgang](https://github.com/tomwolfgang)                                                                                                                                                               | Dev portal - software catalog, tech-docs, scaffolding                                                                                                                                                                                                                                                                                                |
+| [Hotmart](https://www.hotmart.com)                                       | [@fabioviana-hotmart](https://github.com/fabioviana-hotmart)                                                                                                                                                 | The main Developers Portal to centralize docs, applications and technical metrics.                                                                                                                                                                                                                                                                   |
```

Comment:
not sure, why this is happening. the ADOPTERS file should have been prettified, when merged.

Reply:
Think I've fixed this up for you and reset the `ADOPTERS` file from `origin/master`

Reply:
Thanks! :)